formats/schemas/june-2016.txt

```diff
@@ -0,0 +1,77 @@
+Changes for June 2016
+=====================
+
+
+The list of the key changes for the June 2016 major release of the
+OME-XML data model. This schema release will tie in with the Bio-Formats
+5.2 release.
+
+The new major release of the schema has a new namespace and all version
+numbers are reset to 1. As a major release, any file that validated
+correctly using the last major release will probably not validate
+correctly using this new release. Some files that failed to validate
+before will now be valid. It is important to update any file readers and
+writers to understand the changes.
+
+This schema uses the new namespace:
+
+::
+
+    http://www.openmicroscopy.org/Schemas/OME/2016-06/
+
+All the schema files have been unified into a single :file:`ome.xsd` file
+located at this namespace. The version number of this schema file is 1.
+
+Overview of changes
+-------------------
+
+- This release will be implemented in Bio-Formats 5.2 and OMERO 5.3.
+- This release introduces the concept of Folders, a new model object which may
+  contain Images, ROIs and other Folders, and which has a strict tree
+  hierarchy.
+
+
+Details
+^^^^^^^
+
+- The model element ``Folder`` was added. A Folder specifies a possibly
+  heterogeneous collection of data and may contain other Folders, Images and
+  ROIs. Data may be in multiple Folders but a Folder may not be in more than
+  one other Folder, giving a strict tree hierarchy. The primary driver for
+  this addition is to allow the organization of ROI elements into hierarchical
+  structures (see this `Folders blog post <http://blog.openmicroscopy.org/data-model/future-plans/2016/05/23/folders-upcoming/>`_ for further discussion).
+- The ROI properties ``ROI.Namespace``, ``Shape.Linecap`` and
+  ``Shape.Visible`` have been dropped. This simplifies graphical aspects in
+  the data model in favor of more generic enumerated values independent of the
+  rendering framework, making it easier to implement across clients (see this
+  `Design issue <https://github.com/openmicroscopy/design/issues/19>`_ for an
+  example discussion).
+- The (Poly)Line Markers ``Circle`` and ``Square`` have also been dropped for
+  similar reasons to the ROI properties.
```

Comment:
Went over the details of this page with @chris-allan @emilroz and @knabar.  This line should be amended to be more specific and explain that: 
1. the `Marker` enumeration is reduced to `Arrow` only
2. the exact name of the affected attributes `Line.MarkerStart`, `Line.MarkerEnd`, `Polyline.MarkerStart` and `Polyline.MarkerEnd`